airflow/dags/reporting_database_generation/generate-reporting-db.py

```diff
@@ -49,43 +47,63 @@ class SQLTemplatedPythonOperator(PythonOperator):
     'retries': 0,
     'retry_delay': timedelta(minutes=5),
     'concurrency': 4,
-    'queue': 'disambiguator'
-    # 'pool': 'backfill',
-    # 'priority_weight': 10,
-    # 'end_date': datetime(2016, 1, 1),
+    'queue': 'data_collector'
 }
 
 # REPORTING DB
 
 reporting_db_dag = DAG("reporting_database_generation"
                        , default_args=default_args
-                       , start_date=datetime(2022, 6, 30)
-                       , schedule_interval=None
+                       , start_date=datetime(2022, 1, 1)
+                       , end_date=datetime(2022, 7, 1)
+                       , schedule_interval='@quarterly'
                        , template_searchpath="/project/reporting_database_generator/")
 
+assignee_disambiguation_finished = ExternalTaskSensor(
+    task_id="assignee_disambiguation_finished",
+    external_dag_id="inventor_assignee_disambiguation",
+    external_task_id="qc_post_process_assignee",
+    timeout=600,
```

Comment:
How does this work with start/end date? Will this reschedule infinitely until the corresponding task in the other dag is complete?

Reply:
I guess this one you'll still need to verify how it works? Happy to resolve this and approve if you are clear on 

1. which execution date of the inventor/assignee disambiguation dag would this dag depend on ? if this dag is running for 2023/10/01 dag will it automatically know to depend on the 2023/10/01 version (which may be in progress) of assignee/inventor disambiguation instead of, say, 2023/07/01 version (which would be long done)
2. In case when it runs and notices the other dag haven't finished, how long will it wait?

Reply:
My reading of the documentation says that the default is to match each DAG logical date which should be the same between ass-inventor disambiguation and the reporting DB. Is testing this further a requirement for approval?